indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -457,10 +447,16 @@
 
       this.listenTo(view, 'deleted', this.threadDeleted);
       this.listenTo(view, 'closed', this.threadClosed);
-      this.listenTo(view, 'resized', this.layout);
+
+      view.laItemChangedCb = () => {
```

Comment:
my aim here is to setup a listener for `la-gutter-items` so when it goes `active`  the `html` `action-thread-number-container` can be updated, the problem I am facing at the moment is where to setup this up. the view DocumentView renders twice from what I can see